### DISCUSSION TITLE
Observable should be replaced by PropertyChangeListener

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#931] [General]
As the [Observable ](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Observable.html)pattern is deprecated since Java 9, when possible, it should be replaced with the [PropertyChangeListener ](https://docs.oracle.com/en/java/javase/11/docs/api/java.desktop/java/beans/PropertyChangeListener.html)pattern (example [here](https://www.baeldung.com/java-observer-pattern#PropertyChangeListener))

I have done the first replacement in my last [PR](https://github.com/mytourbook/mytourbook/pull/911) and the final result can be seen [here](https://github.com/mytourbook/mytourbook/blob/main/bundles/net.tourbook/src/de/byteholder/geoclipse/poi/GeoQuery.java)

**Comment by wolfgang-ch** [comment]
All Observable's are now removed, fixed in https://github.com/mytourbook/mytourbook/commit/8ce63b42f10f292b01098a5148867e6bd4f10e29 which will be merged into 23.next
